src/main/java/com/autotune/analyzer/serviceObjects/BulkJobStatus.java

```diff
@@ -144,12 +146,17 @@ public void setTotal_experiments(int total_experiments) {
         this.total_experiments = total_experiments;
     }
 
-    public synchronized int getProcessed_experiments() {
+
+    public synchronized void incrementProcessed_experiments() {
```

Comment:
@msvinaykumar - Do we need synchronized on the method as incrementAndGet() itself is atomic?

Reply:
True let me remove it

Reply:
Done